api/src/exam-environment/schemas/exam-generate.ts

```diff
@@ -0,0 +1,22 @@
+import { Type } from '@fastify/type-provider-typebox';
+import { STANDARD_ERROR } from '../utils/errors';
+
+export const examEnvironmentPostExamGenerate = {
+  body: Type.Object({
+    examId: Type.String()
+  }),
+  headers: Type.Object({
+    'exam-environment-authorization-token': Type.String()
+  }),
+  response: {
+    200: Type.Object({
+      data: Type.Object({
+        exam: Type.Record(Type.String(), Type.Unknown()),
+        examAttempt: Type.Record(Type.String(), Type.Unknown())
```

Comment:
Unknown seems a little weird, given that it's a response we're creating. If it's in flux, maybe a TODO to define it?

Reply:
Yes, a lot of this would be nice to properly type, but typebox did not want to play ball. So, this is the interim.